whelk-core/src/main/groovy/whelk/util/LegacyIntegrationTools.groovy

```diff
@@ -101,6 +102,7 @@ class LegacyIntegrationTools {
         return uri
     }
 
+    // FIXME: de-KBV/Libris-ify
```

Comment:
It's a bit of a tangled mess. (I'd go so far as to say we might do it wrong, since, a,ong other things, imported records don't properly track their source records. And record ID handling in MARC systems deal with this, if at all, differently.)